src/gui/search/searchwidget.cpp

```diff
@@ -242,12 +241,18 @@ QTabWidget *SearchWidget::searchTabs() const
 void SearchWidget::on_searchButton_clicked()
 {
     if (Utils::Misc::pythonVersion() < 0) {
-        m_mainWindow->showNotificationBaloon(tr("Search Engine"), tr("Please install Python to use the Search Engine."));
+        // unconditional notifications, user can not disable it
+        Notifications::Request()
+            .title(tr("Search Engine"))
+            .message(tr("Please install Python to use the Search Engine."))
+//          .setWidget(this)
+            .severity(Notifications::Severity::Error)
+            .exec();
```

Comment:
The invocation of the notification here is direct because this notification has to be shown unconditionally. As you may see, all notifications, supplied via `Notifications::Manager::supportedNotifications()` may be disabled by user. I did not want to complicate that code by adding an option to suppress disabling the notification.